File: aoj/volume100/csharp/A10031.cs
using System;
using System.Linq;

class AOJ
{
	static void Main()
	{
		Console.ReadLine(); // Skip 'n'
		var s = Console.ReadLine().Split()
			.Select(w => int.Parse(w));

		Console.ReadLine(); // Skip 'q'
		var t = Console.ReadLine().Split()
			.Select(w => int.Parse(w));

		var c = s.Intersect(t).Count();
		Console.WriteLine(c);
	}
}

File: aoj/volume100/scala/A10031.scala
object Main {
  def main(args: Array[String]) {
    scala.io.StdIn.readLine // to skip n
    val s = (scala.io.StdIn.readLine split " ").map(_.toInt)
    scala.io.StdIn.readLine // to skip q
    val t = (scala.io.StdIn.readLine split " ").map(_.toInt)

    val intersect = s intersect t
    println(intersect.length)
  }
}
